### DISCUSSION TITLE
Anybody know how to add multiiple cur steps That Would really help for my song i made for it

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1548] [Q&A]

**Comment by oketariq** [comment]
why do you need multiple cursteps.. its a variable and if u were to make more they'd all be the same

**Comment by ghost** [comment]
i need multiple cur steps because i have a song that will need multiple things like shake cam

**Comment by Bloopsdoob** [comment]
just
make it run every frame
